packages/ui/src/i18n/dictionaries/authenticator/pt.ts

```diff
@@ -3,13 +3,16 @@ import { AuthenticatorDictionary } from './types';
 export const ptDict: AuthenticatorDictionary = {
   'Account recovery requires verified contact information':
     'A recuperação da conta requer informações de contato verificadas',
+  'Add your Profile': 'Add your Profile',
+  'Add your Website': 'Add your Website',
   'Back to Sign In': 'Voltar para Entrar',
   'Change Password': 'Mudar senha',
   Changing: 'Mudando',
   Code: 'Código',
   'Confirm Password': 'Confirme a Senha',
   'Confirm Sign Up': 'Confirmar inscrição',
   'Confirm SMS Code': 'Confirme o código SMS',
+  'Confirm MFA Code': 'Confirm MFA Code',
```

Comment:
Should we look up these translations as well? seems like it was a miss

Reply:
Oh I missed translating this one! Will fix asap!